CHANGELOG.md

```diff
@@ -3,6 +3,160 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 4.0.0-alpha.38
+
+_Jul 23, 2021_
+
+Big thanks to the 6 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Introduce row editing feature (#2098) @m4theushw
```

Comment:
Could we develop more on this new feature? I think that we could, link the docs, and likely show a screen recording to make it very clear on what's about.

Reply:
A link was added